### DISCUSSION TITLE
Welcome to FidoCadJ Discussions!

### DISCUSSION THREAD

**Discussion by DarwinNE** [FidoCadJ/FidoCadJ#229] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

Discussions are a mean to introduce a subject in an informal way before opening an Issue if there is a bug or something that requires it. The official language is english. Always be polite!

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by manufino** [comment]
@DarwinNE, I’m using this opportunity to ask you what you expect for the future of FidoCadJ. What do you think are the essential steps we should take in the next releases? What features did you have in mind? 
I know you wanted to increase the resolution of the coordinate system, that's no longer a secret :-)
Besides this, have you been scheming anything else over the years?

**Reply by DarwinNE** [reply]
Hi,
   to say the truth, for the moment FidoCadJ covers almost all my needs (mainly schematics for handouts and publications).

Most of ideas I had are in the Issues. For instance, something I'd like to have is this one: https://github.com/DarwinNE/FidoCadJ/issues/210

You may also search for the tag "help wanted": https://github.com/DarwinNE/FidoCadJ/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22

In general, I do not think FidoCadJ should become an overly complicated program, but it should just be doing its job very well.
As a user, I do not particularly like the zoom system. I'd like to be able to use gestures on the trackpad (I do not use mice). I implemented a pinch-style zoom in the (now obsolete) Android application that worked very well, I'd like to have something on the Swing application, too instead of the 1990-style "lens" tool.

Moreover, I do not feel that FidoCadJ is now particularly well integrated in macOS (open files, drag and drop). To say the truth, it somehow pissing me off, as I already solved all those issues in the past and they broke at a certain point.

I also think that FidoCadJ could have been used more in forums and discussion groups in Italy and abroad, but I am not actively promoting the software anymore (apart when I am invited in a seminar or similar).
